src/index.js

```diff
@@ -1,10 +1,10 @@
 /* eslint-disable max-classes-per-file */
 import './style.css';
```

Comment:
[Optional] As much as possible, I suggest that you avoid disabling linter checks inline. Most times, they're pointers to best practices.